tests/README.md

```diff
@@ -14,3 +14,8 @@ These tests are usually associated with assessing performance characteristics of
 
 # Prototype tests
 Tests developed during prototyping.
+
+# ModuleNotFoundeError
```

Comment:
1. Typo in the title
2. I'm not sure if this is the best place to put this paragraph... I'm also not sure where would be best to move it though.
Perhaps we should create a new wiki page for it.

Reply:
hi, i talked to cedric and he recommended to put it in a new wiki page called 'toubleshooting' and like it would be the first entry (which we would update along the line). is this good?

Reply:
Yeah so we would have a "troubleshooting" pages with some issues members can consult and we would have a section on that

Reply:
Yeah that sounds like a good idea